community/sigs/02_sig_infra.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "SIG Infra"
+---
+
+__Focus:__ Creating automation for various Eclipse Tractus-X topics like CI or release. 
+
+__Repository:__ [eclipse-tractusx/sig-infra](https://github.com/eclipse-tractusx/sig-infra)
+
```

Comment:
```suggestion

__Project:__ [System Team](https://github.com/orgs/eclipse-tractusx/projects/9)

```

Reply:
Hmm not sure about that. I also thought about it in the beginning, but came to the conclusion, that the "System Team" focus is a bit broader than just "SIG infra". If we could split that to be more dedicated, then I would add that project later on

Reply:
... understand your point here, but guess then we can also show them how we support and which board we use as sig-infra. Keeping that in mind also ok to start without and there could be more afterwards.